performance_regression_test.py

```diff
@@ -161,6 +161,9 @@ def preload_data(self):
                 if self.params.get('round_robin'):
                     self.log.debug('Populating data using round_robin')
                     params.update({'stress_num': 1, 'round_robin': True})
+                if compaction_strategy:
+                    self.log.debug(f'Next compaction strategy will be used {compaction_strategy}')
+                    params.update({'compaction_strategy': compaction_strategy})
```

Comment:
what happens (in Scylla) when we set `'compaction_strategy': ''` ?

Reply:
will be used default or defined in c-s command